src/slack-api.js

```diff
@@ -0,0 +1,43 @@
+const debug = require('debug')('pagerduty-overrides:pagerduty-api');
+const { WebClient } = require('@slack/client');
+const nconf = require('nconf');
+
+function getClient() {
+  const token = nconf.get('SLACK_TOKEN');
+  if (!token) {
+    throw new Error('SLACK token missing in config');
+  }
+  return new WebClient(token);
+}
+
+function sendSlackMessage(channelId, messageText, done) {
+  const web = getClient();
+  web.chat.postMessage({ channel: channelId, text: messageText })
+    .then((res) => {
+      if (res.ok === false) { return done(new Error(`Slack returned an error: ${res.error}`)); }
+      debug('Message sent: ', res.ts);
+      return done();
+    }).catch((err) => {
+      done(err);
+    });
+}
+
+function getUserList(done) {
```

Comment:
This is for mapping Slack user at PD users?

Reply:
@abtris yes, this is to map slack IDs and user emails (to connect PD and Slack users by email)